flexmeasures/api/v1_1/routes.py

```diff
@@ -271,7 +271,7 @@ def post_weather_data():
 def get_prognosis():
     """API endpoint to get prognosis.
 
-    .. :quickref: User; Download prognosis from the platform
+    .. :quickref: Control; Download prognosis from the platform
```

Comment:
Maybe I don't understand the logic behind your chosen categories, but I think this should be "Data" instead of "Control". Similarly for POSTing prognoses.

Reply:
Yes, I'm on the fence with this one. Not sure if we need those labels for a long time and that it matters much. I'll go with your suggestion.